editor/src/components/navigator/left-pane/github-pane/repository-listing.tsx

```diff
@@ -107,27 +125,47 @@ const RepositoryRow = (props: RepositoryRowProps) => {
           : loadingRepos || !props.importPermitted
           ? 'not-allowed'
           : 'pointer',
-        opacity: loadingRepos || !props.importPermitted ? 0.5 : 1,
+        opacity: loadingRepos || (!props.importPermitted && !props.searchable) ? 0.5 : 1,
         '&:hover': {
           background: colorTheme.dynamicBlue.value,
           color: colorTheme.bg1.value,
           svg: { stroke: colorTheme.bg1.value },
+          '.search-button': {
+            color: colorTheme.fg0.value,
+          },
         },
       }}
       onClick={importRepository}
     >
       <div>
         <Ellipsis style={{ maxWidth: 170 }}>{props.fullName}</Ellipsis>
         <span style={{ fontSize: 10, opacity: 0.5 }}>
-          {props.isPrivate ? 'private' : 'public'}
-          {props.updatedAt == null ? null : (
-            <>
-              {' '}
-              &middot; <TimeAgo date={props.updatedAt} />
-            </>
+          {when(
```

Comment:
You can flip the condition with `unless`.

Reply:
https://github.com/concrete-utopia/utopia/pull/5370/commits/adf3413ca5bb934f22a7d70f442156c506eb1496